tests/unit-tests/api/customers.php

```diff
@@ -29,88 +29,90 @@ public function test_register_routes() {
 		$this->assertArrayHasKey( '/wc/v1/customers/batch', $routes );
 	}
 
-    /**
-     * Test getting customers.
-     *
-     * @since 2.7.0
-     */
-    public function test_get_customers() {
-	    wp_set_current_user( 1 );
+	/**
+	 * Test getting customers.
+	 *
+	 * @since 2.7.0
+	 */
+	public function test_get_customers() {
+		wp_set_current_user( 1 );
```

Comment:
One thing we have started doing in other tests (which must have happened since these customer tests originally got written) is to create a new customer in the suite setup:

```
		$this->user = $this->factory->user->create( array(
			'role' => 'administrator',
		) );
```

and then use it in the tests we need auth `wp_set_current_user( $this->user );` -- instead of just assuming ID 1.